src/ORM/Exception/PersistenceFailedException.php

```diff
@@ -20,5 +20,5 @@
 class PersistenceFailedException extends Exception
 {
 
-    protected $_messageTemplate = 'Failed to %s an entity from %s with primary key "%s" because of: %s';
+    protected $_messageTemplate = 'Entity %s failure';
```

Comment:
I did not want to officially propose an policy or guideline here. But if you guys think it's worth following, why not? 😼